teuthology/results.py

```diff
@@ -82,6 +83,7 @@ def results(archive_dir, name, email, timeout, dry_run):
             )
     finally:
         generate_coverage(archive_dir, name)
+        Scraper(archive_dir).analyze()
```

Comment:
to ensure this gets run, even if generate_coverage() fails, you could wrap generate_coverage in a try/finally as well